diff --git a/server/streamreader/airplay_stream.cpp b/server/streamreader/airplay_stream.cpp
index dcbebae4..6bebc86e 100644
--- a/server/streamreader/airplay_stream.cpp
+++ b/server/streamreader/airplay_stream.cpp
@@ -254,7 +254,7 @@ void AirplayStream::pipeReadLine()
             {
                 // For some reason, EOF is returned until the first metadata is written to the pipe.
                 // If shairport-sync has not finished setting up the pipe, bad file descriptor is returned.
-                LOG(INFO, LOG_TAG) << "Waiting for metadata, retrying in 2500ms\n";
+                LOG(DEBUG, LOG_TAG) << "Waiting for metadata, retrying in 2500ms\n";
                 wait(pipe_open_timer_, 2500ms, [this] { pipeReadLine(); });
             }
             else
